Sheikh_Shahzeb/rails_1/views/users/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>New Form</title>
+	<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
you don't need to include the basic html markup (<html> <head> <body>) in any of your views.  Check out this file in your rails app 'app/views/layouts/application.html.erb'.  That layout is used in all of your views.  The code you put into your view files is injected into the layout where you see the ```<%= yield %>```